android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
Why is 10.0.2.2 permitted?

Reply:
https://developer.android.com/studio/run/emulator-networking

"The virtual router for each instance manages the 10.0.2/24 network address space. All addresses managed by the router are in the form of 10.0.2.xx, where xx is a number. Addresses within this space are pre-allocated by the emulator or router as follows:

...

10.0.2.2 | Special alias to your host loopback interface (127.0.0.1 on your development machine)"